playbooks/aws/openshift-minion/launch.yml

```diff
@@ -65,5 +70,11 @@
       delay: 10
       with_items: ec2.instances
 
-# Apply the configs, seprate so that just the configs can be run by themselves
-- include: config.yml
+- name: "Configure instances"
```

Comment:
We want launch to run configs. That way as we add new config entries, we don't have to add them to launch. If we do it this way, we always have to manually keep them in sync. So -1 on this change. The include is what we want.